<?xml version="1.0"?>

<launch>

<!-- 
  <node name="velocity_generator" pkg="velocity" type="velocity_generator" output="screen">        
  </node> -->

  <node name="acc_node" pkg="velocity" type="acc_node" output="screen">        
    <remap from="/detected_objs" to="/detected_objs"/>
     <param name="direct_control" value="false"/>    
    <param name = "delay_in_sec" value ="1" />
    <param name = "dt" value ="0.1"/>
    <param name = "lag_tau" value = "0.6"/>
    <param name = "Q_vel" value = "1.0"/> 
    <param name = "Q_dis" value =" 4.0"/>
    <param name = "R_weight" value = "0.5"/>
    <param name = "d_time" value ="5.0"/>
    <param name = "d_safe" value ="10.0"/>
    
    


  </node>


  
</launch>
